src/main/kotlin/org/rust/lang/core/macros/MacroExpansionManager.kt

```diff
@@ -380,7 +379,11 @@ private class MacroExpansionServiceImplInner(
         Files.createDirectories(dataFile.parent)
         dataFile.newDeflaterDataOutputStream().use { data ->
             ExpandedMacroStorage.saveStorage(storage, data)
-            MacroExpansionFileSystem.writeFSItem(data, MacroExpansionFileSystem.getInstance().getDirectory(dirs.expansionDirPath))
```

Comment:
`FileNotFoundException` was thrown here. 
I don't know the root cause of this... I hope we'll find it later